src/transformers/tokenization_utils_base.py

```diff
@@ -1745,6 +1745,7 @@ def _from_pretrained(
         if tokenizer_config_file is not None:
             with open(tokenizer_config_file, encoding="utf-8") as tokenizer_config_handle:
                 init_kwargs = json.load(tokenizer_config_handle)
+            init_kwargs.pop("tokenizer_class", None)
```

Comment:
The content of the tokenzier config is passed to the init, so removing this key which is not useful. This is purely cosmetic as the init accepts any kwargs anyways (so if an old version of Transformers gets a `tokenizer_config.json` with a tokenizer class, it will work without any error).